src/pg/sql/42_observatory_exploration.sql

```diff
@@ -372,16 +372,28 @@ BEGIN
              geom_type::TEXT,
              geom_extra::JSONB,
              geom_tags::JSONB,
-      $1 = ANY(numers) valid_numer,
-      $2 = ANY(denoms) valid_denom,
-      $3 = ANY(timespans) valid_timespan
-      FROM observatory.obs_meta_geom
+             $1 = ANY(numers) valid_numer,
+             $2 = ANY(denoms) valid_denom,
+             CASE WHEN $3 IS NOT NULL AND $3 != '' THEN
+              -- Here we are looking for geometries with: a) geometry timespan or b) numerators linked to that geometries that fit in the
```

Comment:
I've checked, before saying no, and is not possible to use IF. You have to use CASE WHEN THEN END